easybuild/easyconfigs/p/pydantic/pydantic-1.10.13-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonBundle'
+
+name = 'pydantic'
+version = '1.10.13'
+
+homepage = 'https://github.com/samuelcolvin/pydantic'
+description = """Data validation and settings management using Python type hinting."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('hatchling', '1.18.0'),
+    ('maturin', '1.4.0', '-Rust-1.75.0'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+]
+
+sanity_pip_check = True
+use_pip = True
+
+exts_list = [
+    ('typing_extensions', '4.9.0', {
```

Comment:
We have this as an easyconfig already: `('typing-extensions', '4.9.0'),`

Reply:
thanks, updated